roofit/roofitcore/inc/RooVectorDataStore.h

```diff
@@ -597,15 +605,17 @@ class RooVectorDataStore : public RooAbsDataStore {
     ClassDef(CatVector,2) // STL-vector-based Data Storage class
   } ;
   
+  std::vector<RealVector*>& realStoreList() { return _realStoreList ; }
+  std::vector<RealFullVector*>& realfStoreList() { return _realfStoreList ; }
+  std::vector<CatVector*>& catStoreList() { return _catStoreList ; }
+
+  void recomputeSumWeight();
```

Comment:
You can put this in a private doxygen section if you want to make it "less" public like here:
https://github.com/root-project/root/blob/6cb78dcb67252d230024c7690f0590a6088e5803/roofit/roofitcore/inc/RooAbsArg.h#L118-L122

Reply:
Good idea, thanks for that!